src/notebooks/controllers/controllerLoader.ts

```diff
@@ -184,72 +136,69 @@ export class ControllerLoader implements IControllerLoader, IExtensionSyncActiva
     }
 
     private async loadControllersImpl() {
-        this.controllersPromise = (async () => {
```

Comment:
This line was removed, thats pretty much the only difference in this method, git diff isn't working too well here to show the differences